dhall-try/index.html

```diff
@@ -222,18 +225,36 @@ <h1 class="text-center">The Dhall configuration language</h1>
       </div>
     </div>
     <div class="container jumbotron" style="background-color: #c5eccf">
-      <h1 class="display-4">Human-friendly</h1>
-      <p class="lead">Dhall supports comments, multi-line string literals and
-      string interpolation with non-technical users in mind.</p>
-      <textarea id="highlight0"></textarea>
-      <p>You can also automatically remove all indirection in any Dhall code,
-      converting the file to a logic-free normal form for non-programmers to
-      understand.</p>
-      <a href="https://github.com/dhall-lang/dhall-lang/wiki/Programmable-configuration-files"class="btn btn-lg btn-outline-dark bg-light">Learn More <i class="fas fa-book"></i></a>
+      <h1 class="display-4">Don't repeat yourself</h1>
+      <hr class="my-4">
+      <p class="lead">Struggling with configuration drift?  Create a <a href="https://en.wikipedia.org/wiki/Single_source_of_truth">single source of truth</a> you can reference everywhere.</p>
```

Comment:
I hadn't heard the term "configuration drift" before, although I can roughly imagine what it means. Would

    Struggling with inconsistent configuration?

capture the same idea?

Reply:
It's a pretty common term in the DevOps world, but I could link "configuration drift" to a definition like this one:

https://kemptechnologies.com/glossary/configuration-drift/

... or even quote the definition of configuration drift inline within the same section:

> Configuration drift occurs when a standardized group of IT resources, be they virtual servers, standard router configurations in VNF deployments, or any other deployment group that is built from a standard template, diverge in configuration over time. As administrators make changes to one resource that isn’t made to another in the same group, or if a new service is installed on some of them, then configurations can drift. Configuration drift can make it very hard to manage IT systems over time. The Infrastructure as Code methodology from DevOps is designed to combat Configuration Drift and other infrastructure management problems.